package.json

```diff
@@ -18,11 +18,10 @@
     "prepublishOnly": "yarn build",
     "build": "rimraf dist && tsc -p tsconfig.build.json",
     "postbuild": "cp -r src/public dist/src/public",
-    "test:unit": "env TZ=UTC ts-node node_modules/.bin/jest * --maxWorkers=4",
     "test:lint": "tslint -p tsconfig.json -c tslint.json",
     "test:type": "tsc --noEmit",
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest",
-    "test": "yarn test:lint && yarn test:type && yarn test:unit"
+    "test": "yarn test:lint && yarn test:type"
```

Comment:
Why do you remove the unit tests from the test command? That means they're never run